src/components/Form2Preview.css

```diff
@@ -0,0 +1,6 @@
+#form2-preview td {
```

Comment:
I've been putting everything so far under Form.css and Form2.css...  It's probably a better idea if there are things that should not be shared that they go into their own individual file.